### DISCUSSION TITLE
LHstudy/note/notec3/

### DISCUSSION THREAD

**Discussion by giscus** [LH-0124/LHstudy#6] [Announcements]
# LHstudy/note/notec3/

https://lh-0124.github.io/LHstudy/note/notec3/

**Comment by yeggwithpassion** [comment]
char不是1个字节嘛

**Reply by LH-0124** [reply]
感谢指正，后续会更正

**Comment by IIIILing** [comment]
这个友链是怎么做的,感觉很带上头像很nice.lhgg是只进行了`mkdocs gh-deploy`吗,github只找到gh分支,没有main分支看不到markdown的源码.

**Reply by LH-0124** [reply]
哈哈哈我是搜到了一个在mkdocs配置友链的一个html代码

**Comment by Sluggish-Boy** [comment]
给lhgg点赞
